nettrine/README.md

```diff
@@ -2,16 +2,35 @@
 
 This is example of nettrine/orm
 
-## Installation
+## Installation with Docker:
```

Comment:
Cim nestacil puvodni setup s PHP dev serverem?

Reply:
V jednoduchosti je sila, nekomplikujme to dockerem prosim.

Reply:
smažu z readme